mllib/src/main/scala/org/apache/spark/ml/Model.scala

```diff
@@ -18,6 +18,7 @@
 package org.apache.spark.ml
 
 import org.apache.spark.ml.param.ParamMap
+import org.apache.spark.util.SizeEstimator
```

Comment:
This is my first time knowing SizeEstimator to estimating the java object size. That's cool.